hw_17/mocha-test/calculator.spec.ts

```diff
@@ -0,0 +1,30 @@
+import assert from 'assert';
+import { expect } from 'chai';
+import { Calculator } from '../src/calculator';
+
+let calculator: any;
+describe('Tests for calculator', () => {
+  before('Initialize class instance', () => {
+    calculator = new Calculator();
+  });
+
+  it('Should check the addition of numbers', () => {
+    assert.equal(calculator.addition(4, 2), 6, 'Addition is not as expected!');
+  });
+
+  it('Should check the subtration of numbers', () => {
+    assert.equal(calculator.subtration(4, 2), 2);
+  });
+
+  it('Should check the multiplication of numbers', () => {
+    assert.equal(calculator.multiplication(4, 2), 8);
+  });
+
+  it('Should check the division of numbers', () => {
+    assert.equal(calculator.division(4, 2), 2);
+  });
```

Comment:
You have very few tests in your suite. Remember: you should write tests using test design technics.
For example, for `addition()` function you should check cases of two positive numbers, two negative numbers, one negative and one positive numbers, the case when one of the arguments is `0`, etc.